hack/tools/tools.go

```diff
@@ -23,12 +23,12 @@ package tools
 import (
 	_ "github.com/drone/envsubst/v2/cmd/envsubst"
 	_ "github.com/golang/mock/mockgen"
-	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
```

Comment:
why are we removing this from here?

Reply:
This is removed because the `golangci-lint` dependency has been removed from `hack/tools/go.mod`